client/src/main/scala/com/dwolla/cloudflare/domain/model/accounts/AccountMember.scala

```diff
@@ -2,6 +2,8 @@ package com.dwolla.cloudflare.domain.model.accounts
 
 case class AccountMember (
   id: String,
+  accountId: String,
```

Comment:
Do we expect to be dealing with multiple account IDs? I wonder if this is better left as a global config value.

Reply:
We have at least two accounts, the account managing hydragents.xyz and the account managing dwolla.com. The account id is passed into each method in the AccountsClient, so it may not be needed on the object. It would be needed to generate the URI for the AccountMember. I was trying to keep all knowledge of generating the URI in this library, and not let it leak into the lambda (or anything else using this lib).

Reply:
This might be a situation where the reader monad could come in handy? Let's chat offline.